chainer/functions/activation/softmax.py

```diff
@@ -2,7 +2,9 @@
 
 import chainer
 from chainer import cuda
-from chainer import function
+from chainer import function_node
+from chainer.functions.array import broadcast
```

Comment:
Use `chainer.functions` namespace instead (same for the next line).

Reply:
Replaced!